index.js

```diff
@@ -159,7 +159,8 @@ Client.prototype.validateXmlrpcVersion = function( callback ) {
 			if ( !error.code ) {
 				return callback( new Error( "Unknown error. " +
```

Comment:
Don't create a new error, that's hiding the information. Just append the text, without "Unknown error" to the existing `message` property.